src/index/base.h

```diff
@@ -5,6 +5,8 @@
 #ifndef BITCOIN_INDEX_BASE_H
 #define BITCOIN_INDEX_BASE_H
 
+#include <string>
```

Comment:
We place system header includes after ones to our codebase.

Reply:
Are there any written guidelines on this? I couldn't find anything in the developer notes and noticed both conventions being used, e.g. `blockfilter.h` for the opposite. Browsing through some more header files, it seems _most_indeed include system headers last, so I'll update it.

Reply:
Could be a good addition to https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#source-code-organization if it's generally-agreed practice

Reply:
Agreed, would be a good addition. Updated all 4 index header files to import system headers last.